mod.ts

```diff
@@ -0,0 +1,3 @@
+import deno from './src/deno.ts'
+
+export default deno
```

Comment:
Just something here. If you export it as deno, the auto import will import it as `deno` on the other side. I suggest keeping the same name

Reply:
I'm exporting it as default
It'll be imported as `import sugar-env from 'https://...'`